fragments/master.sh

```diff
@@ -118,68 +164,38 @@ start_k8s(){
 
     sleep 5
     # Set flannel net config
-    docker -H unix:///var/run/docker-bootstrap.sock run \
+    docker run \
         --net=host ${ETCD_IMAGE} \
         etcdctl \
         set /coreos.com/network/config \
             '{ "Network": "10.1.0.0/16", "Backend": {"Type": "vxlan"}}'
 
     # iface may change to a private network interface, eth0 is for default
-    flannelCID=$(docker -H unix:///var/run/docker-bootstrap.sock run \
-        --restart=on-failure \
+    docker run \
+        --restart=${RESTART_POLICY} \
         -d \
         --net=host \
         --privileged \
         -v /dev/net:/dev/net \
+        -v ${FLANNEL_SUBNET_TMPDIR}:/run/flannel \
```

Comment:
we should keep default location. Are we aware of the reason for the tmp directory?